test/common/network/listen_socket_impl_test.cc

```diff
@@ -9,10 +9,18 @@ TEST(ListenSocket, All) {
   listen(socket1.fd(), 0);
   EXPECT_EQ(15000U, socket1.localAddress()->ip()->port());
 
-  EXPECT_THROW(Network::TcpListenSocket socket2(uint32_t(15000), true), EnvoyException);
+  TcpListenSocket socket2("tcp://127.0.0.1:15002", true);
+  listen(socket2.fd(), 0);
```

Comment:
Do you want to EXPECT on the return value for listen? Same above, I know this was an issue in the original test.